packages/addons/tools/system-tools/package.mk

```diff
@@ -110,7 +110,7 @@ addon() {
 
     # jq
     cp -P $(get_install_dir jq)/usr/bin/jq $ADDON_BUILD/$PKG_ADDON_ID/bin
-    cp -P $(get_install_dir oniguruma)/usr/lib/libonig.so $ADDON_BUILD/$PKG_ADDON_ID/lib
+    cp -P $(get_install_dir oniguruma)/usr/lib/libonig.so* $ADDON_BUILD/$PKG_ADDON_ID/lib
```

Comment:
better just copy what is really necessary + avoid copy new stuff in future that nobody needs
example
`cp -P $(get_install_dir oniguruma)/usr/lib/{libonig.so,libonig.so.2,libonig.so.12}`

Reply:
Got it - done.